substrate-airgapped/Cargo.toml

```diff
@@ -9,15 +9,17 @@ edition = "2018"
 codec = { package = "parity-scale-codec", version = "1.3.5", default-features = false }
 
 # Substrate (hopefully these can be removed in the future)
-sp-runtime = { version = "2.0.0", default-features = false }
-sp-core = { version = "2.0.0", default-features = false }
+sp-runtime = { version = "2.0.1", default-features = false }
+sp-core = { version = "2.0.1", default-features = false }
+primitive-types = { version = "0.7", default-features = false, features = ["rustc-hex", "serde"] }
```

Comment:
What is the advantage of using incorporating this crate instead of using the `H256` type from `sp-core`? Even if `sp-core` re-exports I prefer to just keep one dependency because I think it makes auditing the dependency trees easier (correct me if I am wrong though). Also, seems like this dependency is only used in the examples so I would prefer to add it as a dev dependency for now

Reply:
Yeah, was on the fence myself. It's here because I wanted to use `H256::from_str` rather than reimplement it. That requires the `rustc-hex` feature which isn't available through `sp-core` (I think?), but you're right that it's a bit weird to import it twice.